templates/functions.php

```diff
@@ -35,6 +35,11 @@ function ap_scripts_front() {
 	wp_add_inline_style( 'anspress-main', $custom_css );
 	do_action( 'ap_enqueue' );
 
+	// Enqueue the TinyMCE JavaScript library file.
+	if ( ap_opt( 'question_text_editor' ) || ap_opt( 'answer_text_editor' ) ) {
```

Comment:
@mebishalnapit Check my comment on old PR.

Reply:
@rahularyan I've replied to the old comment. That way, tinyMCE failed to get initialized on page load and hence the issue remained. So, had to resolve it in this way.

Reply:
@mebishalnapit, unfortunately, this approach isn't viable. Our objective is to avoid loading TinyMCE on every page load. It appears to be initialized only upon clicking the editor. This intentional delay in loading TinyMCE until the dummy editor is clicked serves our purpose.

Reply:
@rahularyan Added additional checks for single question page and ask page to enqueue TinyMCE.